brave/browser-laptop
# Updates bat libraries

Resolves #13509 
Resolves #13431 (https://github.com/brave-intl/bat-publisher/pull/24)

## Submitter Checklist:


## Test Plan:
1) clean profile
2) enable payments
3) visit some sites
4) pin few sites
5) trigger contribution (close browser, change reconcile timestamp, run with `LEDGER_NO_DELAY=true LEDGER_NO_FUZZING=true LEDGER_VERBOSE=true` params)
6) make sure that pinned sites are in the same ball park (could be couple % off of course)
7) open `ledger-state.json` and make sure that there is memo object in there

## Reviewer Checklist:


Tests
